src/ast/attachpoint_parser.cpp

```diff
@@ -778,5 +781,24 @@ AttachPointParser::State AttachPointParser::iter_parser()
   return OK;
 }
 
+AttachPointParser::State AttachPointParser::raw_tracepoint_parser()
+{
+  if (parts_.size() != 2)
+  {
+    if (ap_->ignore_invalid)
+      return SKIP;
+
+    errs_ << ap_->provider << " probe type requires 1 argument" << std::endl;
+    return INVALID;
+  }
+
+  ap_->func = parts_[1];
+
+  if (ap_->func.find('*') != std::string::npos)
+    ap_->need_expansion = true;
```

Comment:
If we want to support wildcards, we should probably add tests for them. Perhaps it would be easier to omit wildcard support for now and add it as a follow-up PR?